### DISCUSSION TITLE
Loading spinner acts weird while on spotlight rankings

### DISCUSSION THREAD

**Discussion by TheRealStevie** [ppy/osu#24473] [Q&A] [answered]
Sometimes the loading spinner gets tiny or it does not appear while interacting in spotlight rankings.

https://github.com/ppy/osu/assets/75444413/e214c268-a48b-4abf-b2cf-7b4f0c9a7483

[database.log](https://github.com/ppy/osu/files/12269055/database.log)
[input.log](https://github.com/ppy/osu/files/12269056/input.log)
[network.log](https://github.com/ppy/osu/files/12269057/network.log)
[performance.log](https://github.com/ppy/osu/files/12269059/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12269060/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12269061/updater.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24479
